README.md

```diff
@@ -1,2 +1,10 @@
 # MV-Trial-Project-Review
 A review of my microverse trial project
+
```

Comment:
Kindly ensure you add a table of contents in your README for easy navigation of the README.md file.